images/toolchain-arm-linux-gnueabihf/hooks/images.csv

```diff
@@ -0,0 +1,4 @@
+DOCKERFILE;ARGS (comma-separated list);TAG
+;MAJOR=7.4;latest
+;MAJOR=4.9;4.9
```

Comment:
The tag should be `gcc-4.9`, not `4.9` (we should preserve compatibility with existing tags).

Also we should add `gcc-7.4`  tag.